src/lando/main/scm/__init__.py

```diff
@@ -1,10 +1,61 @@
-# flake8: noqa
-from lando.main.scm.abstract_scm import *
-from lando.main.scm.consts import *
-from lando.main.scm.exceptions import *
-from lando.main.scm.hg import *
+from lando.main.scm.abstract_scm import AbstractSCM
+from lando.main.scm.consts import (
+    SCM_GIT,
+    SCM_HG,
+)
+from lando.main.scm.exceptions import (
+    AutoformattingException,
+    NoDiffStartLine,
+    PatchApplicationFailure,
+    PatchConflict,
+    SCMException,
+    SCMInternalServerError,
+    SCMLostPushRace,
+    SCMPushTimeoutException,
+    TreeApprovalRequired,
+    TreeClosed,
+)
+from lando.main.scm.git import GitSCM
+from lando.main.scm.hg import (
+    REQUEST_USER_ENV_VAR,
+    HgCommandError,
+    HgException,
+    HgSCM,
+    hglib,
+)
 
+# These can only be determined when all the subclasses of the AbstractSCM have been defined.
+SCM_CHOICES = {
+    klass.scm_type(): klass.scm_name() for klass in AbstractSCM.__subclasses__()
+}
 SCM_IMPLEMENTATIONS = {
-    # SCM_GIT: GitSCM,
-    SCM_HG: HgSCM,
+    klass.scm_type(): klass for klass in AbstractSCM.__subclasses__()
 }
```

Comment:
@cgsheeh Here's my suggestion to address your comment in https://github.com/mozilla-conduit/lando/pull/154#discussion_r1866870640